assets/models/menu_backgrounds/MainMenuScene1.tscn

```diff
@@ -374,3 +415,5 @@ lifetime = 1.42
 preprocess = 2.08
 mesh = SubResource("38")
 gravity = Vector3(0, 0, 0)
+
+[editable path="MenuPlanet"]
```

Comment:
Would it be possible to fix the planet scene so that at least the space or the main menu scenes don't need to modify it? Right now it looks like both of the places that use the planet need to modify it to make it work, so the planet scene is hard to use right now if the default settings of it are unusable.

Reply:
I was just working on that. I think part of the issue in how it was setup was due to the way Godot uses .glb scenes, which adds complexity. I'm switching to using two .mesh files for the planet layers - one basic sphere, and one with the normals flipped to use for the atmosphere (so it doesn't show in front of the planet). I've also made the planet in the menu scene local to there, to remove overlap. I wasn't sure what that specific _editable path_ line does.

Reply:
It makes instanced scene child nodes editable. Which is fine for one-off customizations, but if the customizations are needed in multiple places. Then it is better to make one derived scene from the base scene, apply the customizations there, and then instantiate that derived scene where it is needed. This way the manual scene customizations only need to be in a single place.